kodilla-spring/src/test/java/com/kodilla/spring/portfolio/BoardTestSuite.java

```diff
@@ -0,0 +1,26 @@
+package com.kodilla.spring.portfolio;
+
+import org.junit.jupiter.api.Test;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.context.ApplicationContext;
+import org.springframework.context.annotation.AnnotationConfigApplicationContext;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+
+@SpringBootTest
```

Comment:
Jesli uzywasz AnnotationConfigApplicationContext, to ta adnotacja jest niepotrzebna